scripts/post-install.sh

```diff
@@ -53,6 +50,12 @@ if [[ -f /etc/redhat-release ]]; then
         install_chkconfig
     fi
 elif [[ -f /etc/debian_version ]]; then
+    # Ownership for RH-based platforms is set in build.py via the `rmp-attr` option.
+    # We perform ownership change only for Debian-based systems.
```

Comment:
Do these options also need to be added to the Amazon Linux block at line 69?

Reply:
Never mind, Amazon Linux and Amazon Linux are both CentOS-based with yum. Might be worth manually testing though.